mycroft/client/speech/main.py

```diff
@@ -56,6 +56,10 @@ def handle_utterance(event):
     ws.emit(Message('recognizer_loop:utterance', event))
 
 
+def handle_unknown():
+    ws.emit(Message('speech.recognition.unknown'))
```

Comment:
@penrods suggested using recognizer_loop:x internally in the speech client and using mycroft.x.y on the interprocess message bus.

Reply:
OK, this has been changed to `recognizer_loop:speech.recognition.unknown` and `mycroft.speech.recognition.unknown`. Let me know if any other name sounds better.